server/mongo_data_access.go

```diff
@@ -52,6 +100,11 @@ func (dal *mongoDataAccessLayer) PostWithID(id string, resource interface{}) err
 	resourceType := reflect.TypeOf(resource).Elem().Name()
 	collection := dal.Database.C(models.PluralizeLowerResourceName(resourceType))
 	updateLastUpdatedDate(resource)
+
+	if dal.hasInterceptorsForVerbAndType("POST", resourceType) {
```

Comment:
I see what you mean. I really intended the hasInterceptorsForVerbAndType() to prevent extra work for the DELETE operations anyway.